Content.Client/UserInterface/GameHud.cs

```diff
@@ -344,110 +436,215 @@ public bool SandboxButtonVisible
+                    {
+                        name = null;
+                        return false;
+                    }
+
+                    name = null;
+                    name = key switch
+                    {
+                        Apostrophe => "'",
+                        Comma => ",",
+                        Delete => "Del",
+                        Down => "Dwn",
+                        Escape => "Esc",
+                        Equal => "=",
+                        Home => "Hom",
+                        Insert => "Ins",
+                        Left => "Lft",
+                        Menu => "Men",
+                        Minus => "-",
+                        Num0 => "0",
+                        Num1 => "1",
+                        Num2 => "2",
+                        Num3 => "3",
+                        Num4 => "4",
+                        Num5 => "5",
+                        Num6 => "6",
+                        Num7 => "7",
+                        Num8 => "8",
+                        Num9 => "9",
+                        Pause => "||",
+                        Period => ".",
+                        Return => "Ret",
+                        Right => "Rgt",
+                        Slash => "/",
+                        Space => "Spc",
+                        Tab => "Tab",
+                        Tilde => "~",
+                        BackSlash => "\\",
+                        BackSpace => "Bks",
+                        LBracket => "[",
+                        MouseButton4 => "M4",
+                        MouseButton5 => "M5",
+                        MouseButton6 => "M6",
+                        MouseButton7 => "M7",
+                        MouseButton8 => "M8",
+                        MouseButton9 => "M9",
+                        MouseLeft => "ML",
+                        MouseMiddle => "MM",
+                        MouseRight => "MR",
+                        NumpadDecimal => "N.",
+                        NumpadDivide => "N/",
+                        NumpadEnter => "Ent",
+                        NumpadMultiply => "*",
+                        NumpadNum0 => "0",
+                        NumpadNum1 => "1",
+                        NumpadNum2 => "2",
+                        NumpadNum3 => "3",
+                        NumpadNum4 => "4",
+                        NumpadNum5 => "5",
+                        NumpadNum6 => "6",
+                        NumpadNum7 => "7",
+                        NumpadNum8 => "8",
+                        NumpadNum9 => "9",
+                        NumpadSubtract => "N-",
+                        PageDown => "PgD",
+                        PageUp => "PgU",
+                        RBracket => "]",
+                        SemiColon => ";",
+                        _ => DefaultShortKeyName(keyFunction)
+                    };
+                    return name != null;
                 }
+
+                name = null;
+                return false;
             }
```

Comment:
Could be useful to have this elsewhere but I guess if anyone else ends up needing it they can change it.